x-pack/test/saved_object_api_integration/user_profiles/apis/create.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { loginAsInteractiveUser } from '../helpers';
+import { TEST_CASES } from '../../common/suites/create';
+import { AUTHENTICATION } from '../../common/lib/authentication';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertestWithoutAuth');
+  describe('create', function () {
+    let sessionHeaders: { [key: string]: string } = {};
+
+    before(async () => {
+      sessionHeaders = await loginAsInteractiveUser({
+        getService,
+        ...AUTHENTICATION.KIBANA_RBAC_USER,
+      });
+    });
+
+    it('created_by/updated_by is with profile_id', async () => {
+      const soType = TEST_CASES.NEW_SINGLE_NAMESPACE_OBJ.type;
+      const createResponse = await supertest
+        .post(`/api/saved_objects/${soType}`)
+        .set(sessionHeaders)
+        .send({ attributes: { title: 'test' } });
+
+      expect(createResponse.status).to.be(200);
+      const so = createResponse.body;
+      expect(typeof so.created_by).to.be('string');
+      expect(typeof so.updated_by).to.be('string');
```

Comment:
__suggestion:__ I'm wondering if in `loginAsInteractiveUser` you could also retrieve user profile ID like this https://github.com/elastic/kibana/blob/69b28f317b07af12fa346571b04b411a6130b25f/x-pack/test/security_api_integration/tests/user_profiles/get_current.ts#L68-L79 so that here you could really be sure that Kibana populated `created_by` and `updated_by` with user profile ID and not with some random string? It'd especially be useful in `update` tests where objects are updated by different users.

Reply:
thanks, would be a nice improvement, I'll take a look